main.go

```diff
@@ -108,10 +110,20 @@ var (
 
 func init() {
 	_ = scheme.AddToScheme(myscheme)
+
+	// metal3Ipam and capipam schemes
 	_ = ipamv1.AddToScheme(myscheme)
-	_ = caipamv1.AddToScheme(myscheme)
+	_ = capipamv1beta1.AddToScheme(myscheme)
+	_ = capipamv1.AddToScheme(scheme.Scheme)
```

Comment:
(commenting on an old PR)

@adilGhaffarDev why not:

```diff
-_ = capipamv1.AddToScheme(scheme.Scheme)
+_ = capipamv1.AddToScheme(myscheme)
```

Reply:
yes this should be _ = capipamv1.AddToScheme(myscheme) . I will open a pr with the fix.